NOTICE

```diff
@@ -10,10 +10,9 @@ Different parts of CrateDB are available under different licences.
 Unless otherwise stated, every component of CrateDB is licensed under the
 Apache License, Version 2.0.
 
-CrateDB also includes enterprise features. These features are marked in the
-documentation and in their respective source files, and are not licensed under
-the Apache License. To use these components, you must purchase a commercial
-licence agreement with Crate.
```

Comment:
is there a specific reason to remove that?

Reply:
I have replaced it with a rewritten para in the same location

see here: https://github.com/crate/crate/pull/7472/files#diff-aa14599dc3fdfa9cc81a7c3c247b77b9R13

basically what I've done here is standardize on the word "module" to mean "the chunks of code" (as we refer to them as "modules" elsewhere) and then clarify that the modules provide "features" (which we use in our wording of the legal text)

I have then shortened this and ended the para by saying that the modules are not available under an Apache license

I dropped the "To use these components, you must purchase a commercial license agreement with Crate." part, because the very next paragraph repeats the same thing

see https://github.com/crate/crate/blob/b498ded52555d5bb0783518959b6d351535ae8a4/NOTICE#L17

Reply:
got it! thanks.